janusgraph-server/pom.xml

```diff
@@ -88,4 +88,12 @@
             </exclusions>
         </dependency>
     </dependencies>
+    <build>
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/resources</directory>
```

Comment:
I don't quite understand the purpose of this addition. I don't see any resources here under `src/main/resources`.